src/api/general.ts

```diff
@@ -200,11 +200,36 @@ export class General {
    */
   async view<T extends Array<MoveValue>>(args: {
     payload: InputViewFunctionData;
-    options?: LedgerVersionArg;
+    options?: LedgerVersionArg & { bcsOutput?: boolean };
```

Comment:
either we remove it here, or we need to handle it in the `view` function - I think we should just remove it from here

Reply:
Yeah, let me go back and fix these

Reply:
I dont think we do something with the `bcsOutput`?

Reply:
woops, thought I removed this